app/scoring/page.tsx

```diff
@@ -57,16 +58,16 @@ export default async function Scoring() {
 
     const contestantRoundScores: number[] = generateContestantRoundScores(currentSelectedContestantTeamsList, numberOfRounds)
 
+    const listOfContestantRoundLists = [{
+            key: "Jacob",
+            content: <ContestantRoundList perfectRoundScores={roundScores} contestantRoundScores={contestantRoundScores} perfectTeamList={reverseTeamsList} contestantTeamList={currentSelectedContestantTeamsList}/>
+    }]
+
     return (
         <div>
-            <h1 className="text-2xl text-center">Current Scoring for {currentSelectedContestant}</h1>
```

Comment:
I should remove `currentSelectedContestant` from the code since this is no longer used after this.